sdk/fd_create/test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+# This test script is not ready to be released!
```

Comment:
Can you add the copyright/licensing header to this.  Also, why is this file needed rather than e.g. hooking the example into our integration tests?

Reply:
We don't have a C toolchain available and I'm not sure if we want to add that dependency yet.